### ISSUE TITLE
Inconsistent Fed & Hydrated Behavior

### ISSUE THREAD

**Issue by Constellation-VASCO** [Starfield-Community-Patch/Starfield-Community-Patch#966]
The 'Fed' and 'Hydrated' active effects will randomly refresh without eating or drinking.

---

## Game Version
1.11.36.0

## Language
English (EN)

## New Game
Yes

## Reproduction Steps
Enable Sustenance with both Positive and Negative effects. Play normally until you receive an active effect notification 'You are now fed' or 'You are now hydrated' without eating or drinking.

**Comment by Deebz96** [comment]
I believe I see what is causing this. Some of the magic effects related to this are set up as a "Peak Value Modifier" for "_TempAV". In every other case this actor value is referenced, it is set as a "Value Modifier" type.

Been testing out a fix which changes these all to "Value Modifier" as well. Everything still works as expected, and I have yet to see these effects get re-applied. I'll keep testing and post the fix if all seems to be in order.

**Comment by Deebz96** [comment]
After some days of testing, it seems that changing the magic effect type does fix the issue with the effects becoming re-applied randomly. Despite this though, I did still see the "You are now hydrated" message appearing randomly. Just the message though, without the effects becoming re-applied.

Because of this, I have also added conditions to suppress the "You are now fed" and "You are now hydrated" messages if you are already fed/hydrated. This does mean you won't see these messages if you eat or drink before those effects run out, but the effects will still be renewed regardless. This change makes sense to me anyway, as your status is going from Fed to Fed. You are not _only now_ becoming fed.

Probably the best we can do here, until Bethesda hopefully remakes this system into something more proper.

[SusSpamFix.zip](https://github.com/user-attachments/files/16570688/SusSpamFix.zip)
